ci-operator/config/openshift/file-integrity-operator/openshift-file-integrity-operator-master.yaml

```diff
@@ -61,6 +75,28 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-aws
+- as: e2e-aws-upgrade
+  steps:
+    workflow: "optional-operators-ci-aws"
+    cluster_profile: aws
+    env:
+      OO_CHANNEL: "release-v0.1"
+      OO_PACKAGE: "file-integrity-operator"
+      OO_INSTALL_NAMESPACE: '!create'
+      OO_TARGET_NAMESPACES: '!install'
+    dependencies:
+      OO_INDEX: "upgrade-bundle"
+    test:
+    - as: e2e-upgrade
+      cli: latest
+      commands: make e2e
```

Comment:
expecting that we'll need adjustments in make e2e to work against a catalog-installed operator.

Reply:
I started kicking the tires here https://github.com/openshift/file-integrity-operator/issues/269